README.md

```diff
@@ -48,31 +48,22 @@ independent databases gives you.
 
 ## Postgres version
 
-The JAR file contains bundled version of Postgres. You can pass different Postgres version by implementing [`PgBinaryResolver`](src/main/java/com/opentable/db/postgres/embedded/PgBinaryResolver.java).
+It is possible to change postgres version:
```

Comment:
Adapt some of my wording in CHANGELOG. In particular you should note what. no longer exists
- Support for a local postgres (for now)
- any other config options that changed.